src/components/DraggableColumn/DraggableColumn.css

```diff
@@ -0,0 +1,9 @@
+.draggableDivider {
+  left: 0;
+  top: 0;
+  width: 3vh;
```

Comment:
I don't like the idea of the width of this changing based on the height of the viewport. Let's use `px` here.

Reply:
What px value do you think is reasonable?